istio.deps

```diff
@@ -11,6 +11,6 @@
 		"name": "ENVOY_SHA",
 		"repoName": "istio/envoy",
 		"file": "WORKSPACE",
-		"lastStableSHA": "bdb54fa3dc5c6e02a3746c8e2fd6c56640d17799"
+		"lastStableSHA": "d24427d43edfb1bd722aa781e22f87a11ec51e8d"
```

Comment:
Is anything using this file? Can it be nuked?

Reply:
I've seen this discussed many times, but the file is still there. :)

Reply:
And yes, nothing uses this file AFAICT.

Reply:
Ok. It's a house of cards, so I'd rather not disturb it. Who knows what's out there depends on this.